webservices/utils.py

```diff
@@ -13,7 +13,9 @@
 from sqlalchemy.orm import foreign
 from sqlalchemy.ext.declarative import declared_attr
 from sqlalchemy.dialects import postgresql
-from elasticsearch import Elasticsearch
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+import certifi
```

Comment:
Does `certifi` need to be in `requirements.txt`?

Reply:
certifi is a dependency pkg of requests-aws4auth==1.0 which we already have in requirements.txt file. We don't need to to specify again.

Reply:
Because we're using it as a standalone package, let's go ahead and put it in requirements.txt to avoid breaking changes.

Reply:
done!

Reply:
Thanks @pkfec! Interesting versioning `certifi` has - looks like they're just tagging releases with dates and not following semantic versioning...

Thanks for addressing my comments! Looks great.